js/data/feature_tree.js

```diff
@@ -66,15 +66,10 @@ FeatureTree.prototype.query = function(args, callback) {
 };
 
 function geometryContainsPoint(rings, type, p, radius) {
-    if (type === 'Point') {
-        return pointContainsPoint(rings, p, radius);
-    } else if (type === 'LineString') {
-        return lineContainsPoint(rings, p, radius);
-    } else if (type === 'Polygon') {
-        return polyContainsPoint(rings, p) ? true : lineContainsPoint(rings, p, radius);
-    } else {
-        return false;
-    }
+    if (type === 'Point') return pointContainsPoint(rings, p, radius);
```

Comment:
In this particular case it looks much cleaner to me than the original version. Also you don't usually set break points on this type of conditional set since it's comparing the same basic string value. Should I rewrite this as a ternary operator instead?